lib/mutant/mutator/node/binary.rb

```diff
@@ -21,7 +21,11 @@ def dispatch
           emit_singletons
           emit_promotions
           emit_operator_mutations
-          emit_left_mutations
+
+          emit_left_mutations do |mutation|
+            !(n_irange?(mutation) || n_erange?(mutation)) || !mutation.children.fetch(1).nil?
```

Comment:
Endless ranges to the left of an `||` are invalid syntax. Welcome to ruby syntax design.